src/components/accounts/two_factor/TwoFactorVerifyModal.js

```
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { useDispatch, useSelector } from 'react-redux';
+import Modal from "../../common/modal/Modal";
+import ModalTheme from '../ledger/ModalTheme';
+import MobileActionSheet from '../../common/modal/MobileActionSheet';
+import FormButton from '../../common/FormButton';
+import { Translate } from 'react-localize-redux';
+import TwoFactorVerifyInput from './TwoFactorVerifyInput';
+import { verifyTwoFactor, clearAlert, resendTwoFactor, get2faMethod } from '../../../actions/account';
+
+const Form = styled.form`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+`
+
+const TwoFactorVerifyModal = ({ open, onClose }) => {
+
+    const [method, setMethod] = useState();
+    const [code, setCode] = useState('');
+    const dispatch = useDispatch();
+    const account = useSelector(({ account }) => account);
+    const loading = account.actionsPending.includes('VERIFY_TWO_FACTOR');
+
+    useEffect(() => {
+        let isMounted = true;
+
+        const handleGetTwoFactor = async () => {
+            setMethod(await dispatch(get2faMethod()))
+        };
+
+        if (isMounted) {
+            handleGetTwoFactor()
+        }
+        
+        return () => { isMounted = false }
+    }, []);
+
+    const handleVerifyCode = async () => {
+        try {
+            await dispatch(verifyTwoFactor(null, code))
+        } catch(e) {
+            return;
```

why error is ignored?